src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -236,6 +239,12 @@ export class BaseEditCourseForm extends React.Component {
       && parseOptions(courseRunOptionsData.content_language.choices));
     const programOptions = (courseRunOptionsData
       && parseOptions(courseRunOptionsData.expected_program_type.choices));
+    const locationCountryOptions = courseOptionsData
+      && parseOptions(courseOptionsData.location_restriction.children.countries.child.choices);
```

Comment:
Just to confirm, As this is too much nesting so do we need to add `&` to avoid the code break?

Reply:
Do you mean something like
`parseOptions(courseOptionsData?.location_restriction.children.countries.child.choices)`?

if the argument returns false parseOptions actually breaks

Reply:
Oh sorry I meant `?` not `&`. And if parseOptions breaks on undefined then this is good